online-shopping-website/src/Components/ProductDetails.js

```diff
@@ -0,0 +1,128 @@
+import Grid from '@mui/material/Grid';
+import Card from '@mui/material/Card';
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import AddShoppingCartIcon from '@mui/icons-material/AddShoppingCart';
+import Stack from '@mui/material/Stack';
+import AddIcon from '@mui/icons-material/Add';
+import RemoveIcon from '@mui/icons-material/Remove';
+import {useParams} from 'react-router-dom';
+import Products from '../TestValues.json';
+
+class ProductButtons extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            quantity: 1,
+            show: true,
+            max: 5,
+            min: 0
+        };
+    }
+
+    IncrementItem = () => {
+        if (this.state.quantity != 10) {
+            this.setState({
+                quantity: this.state.quantity + 1
+            });
+        }
+    }
+    DecreaseItem = () => {
+        if (this.state.quantity != 1) {
```

Comment:
For `IncreaseItem()` and `DecreaseItem()` please use `!==` instead of `!=`. It's better practice for JS because it typechecks as well as checking for equality. It's a small thing but it's helpful for best practices:)